docs/LOAD-BALANCER.md

```diff
@@ -0,0 +1,29 @@
+Documentation for Load-Balancer:
+
+## Server Setup Instructions:
+* Setup [Nginx setup guide)(https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) for Ubuntu 18.04 on a new EC2 instance. *Supplementary step*: Add [Nginx](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/) PPA to run latest stable Nginx version, instead of the default OS Nginx, since it's not updated as frequently and can lag on some feature/security fixes.
```

Comment:
Your markdown link is borked. Replace closing `)` with `]`.

Reply:
Also just reads weird as "Setup Nginx setup guide..." Maybe just "Setup Nginx..."?